pipeline/frontend/yaml/linter/linter.go

```diff
@@ -275,21 +291,6 @@ func (l *Linter) lintDeprecations(config *WorkflowConfig) (err error) {
 		return err
 	}
 
-	for _, container := range parsed.Steps.ContainerList {
-		if len(container.Secrets) > 0 {
-			err = multierr.Append(err, &errorTypes.PipelineError{
-				Type:    errorTypes.PipelineErrorTypeDeprecation,
-				Message: "Usage of `secrets` is deprecated, use `environment` with `from_secret`",
```

Comment:
We already had proper error messages, why were they removed?

Reply:
we had but just for steps, not clone or serivices ...

so thats why I initially frontport the `lintContainerDeprecations` ...

this codeblock here did not exist in 2.8 and those as i detected it i removed it because of redudancy

-> I never comparted the two messages, through i probably should have